### DISCUSSION TITLE
Repeater won't send values to database when 2 or more fields are inside the repeater.(i've to use same name as the input's name but how can i name them when 2 or more fields are there?)

### DISCUSSION THREAD

**Discussion by isubarnaL** [filamentphp/filament#12639] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

this is my resource code 
```

                Repeater::make('tripplans')
                ->schema([
                    TextInput::make('triptitle')
                    ->required(),
                    TextInput::make('tripdesc')
                    ->required(),
                ])
                ->addActionLabel('Add Another Day')
                ->columns(2)
                ->columnSpan(2),
            
```
if i just make the name of repeater triptitle/tripdesc it will send the data of that named field and won't send input data of other field

**Comment by isubarnaL** [comment]
I've found that when i change the name of the repeater to triptitle, it sends the value of triptitle input as intended but its still not sending the value of tripdesc. how can i write the name of the repeater so as to send the value of all input fields?

**Reply by leandrocfe** [reply]
You should create a`tripplans` column (JSON type) in your database and add an array cast in your Model
